src/components/__tests__/test.test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('../Test');
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
```

Comment:
- [ ] Kindly add tests for the components you created, as well as snapshots. This test here does not represent any of the app components. I have added an image for your reference.

![test](https://github.com/Shaheryar0054/react-group-project/assets/53124548/b5270f7b-aab9-47c2-9503-fd9ee7808db6)